tvApp/src/main/java/band/effective/office/tv/screen/menu/MenuScreen.kt

```diff
@@ -1,86 +1,132 @@
-            Spacer(Modifier.height(20.dp))
-            MenuComponent(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .height(400.dp),
-                itemsList = itemsList.map { Pair(it.screen, ItemRes(text = it.title)) },
-                onNavigate = { navController.navigate(it.name) })
+            when (index) {
+                0 -> {
+                    Text(
+                        modifier = Modifier.width(635.dp),
+                        text = stringResource(R.string.welcome_string),
+                        fontFamily = robotoFontFamily(),
+                        fontSize = 50.sp,
+                        color = Color.White,
+                        textAlign = TextAlign.Center
+                    )
+                    Spacer(Modifier.height(10.dp))
+                    Text(
+                        modifier = Modifier.width(400.dp),
+                        text = stringResource(id = R.string.main_menu_text),
+                        color = Color.White,
+                        style = MaterialTheme.typography.body1,
+                        textAlign = TextAlign.Center
+                    )
+                    Spacer(Modifier.height(20.dp))
+                    PlayButton(
+                        modifier = Modifier
+                            .height(50.dp)
+                            .width(210.dp)
+                            .clip(shape = RoundedCornerShape(40.dp))
+                            .onFocusChanged { isFocusOnMenu = !it.isFocused }
+                            .focusRequester(focusRequester),
+                        text = stringResource(id = R.string.autoplay_button),
+                        onFocus = { coroutineScope.launch { scrollState.animateScrollTo(0) } },
+                        onClick = { navController.navigate(Screen.Autoplay.name) })
+                    Spacer(Modifier.height(5.dp))
+                    ButtonAutoplay(
+                        modifier = Modifier
+                            .height(50.dp)
+                            .width(190.dp)
+                            .clip(shape = RoundedCornerShape(210.dp))
+                            .onFocusChanged { isFocusOnMenu = !it.isFocused },
+                        onFocus = { coroutineScope.launch { scrollState.animateScrollTo(0) } },
+                        onClick = { navController.navigate(Screen.AutoplayMenu.name) }) {
+                        Text(
+                            text = stringResource(id = R.string.autoplay_settings),
+                            color = Color.White,
+                            modifier = Modifier
+                                .alpha(if (it) 1f else 0.5f),
+                            style = MaterialTheme.typography.body1
+                        )
+                    }
+                }
+
+                1 -> {
+                    MainMenuPlaceHolder("Раздел фото")
```

Comment:
string.xml

Reply:
it's temp placeholder, does it make sense to put this into a resource?